### PR TITLE
Task/backend204 implement replies for comment

### PR DESCRIPTION
**Pull request by droztoff**

### CODE REVIEW

**Comment by droztoff** [comment]
[The preview from the previous video page is displayed after opening the new video page](https://trello.com/c/G7FSYZUM/247-the-preview-from-the-previous-video-page-is-displayed-after-opening-the-new-video-page)

**Comment by droztoff** [comment]
[Implement the possibility to comment the comments of other users on the Video page](https://trello.com/c/0om6bEPz/204-implement-the-possibility-to-comment-the-comments-of-other-users-on-the-video-page)

File: backend/core/prisma/schema.prisma [hunk 1]
```diff
@@ -133,6 +133,7 @@ model CategoryToUser {
 
 model VideoComment {
   id               String            @id @default(uuid())
+  parentId         String?
```
Comment (WanderningMaster) [inline backend/core/prisma/schema.prisma:136]: optional string columns should be non-null and defaulted to (""), see comments in scheme.prisma file

File: backend/core/src/secondary-adapters/postgres/video/video-repository-adapter.ts [hunk 2]
```diff
@@ -70,15 +75,21 @@ export class VideoRepositoryAdapter implements VideoRepository {
...
       return video;
     }
+
     const { dislikeNum, likeNum } = await this.calculateReaction(video.id);
-    return {
-      ...trimVideoWithComments(video),
-      likeNum,
-      dislikeNum,
-    };
+    const commentsWithReplies = await Promise.all(
```
Comment (WanderningMaster) [inline backend/core/src/secondary-adapters/postgres/video/video-repository-adapter.ts]: I guess its better to use prisma transaction api in this case

**Review by WanderningMaster — CHANGES_REQUESTED** [review]

**Review by WanderningMaster — APPROVED** [review]

**Pull request merged**